pages/_document.tsx

```diff
@@ -10,6 +10,7 @@ export default function Document(): JSX.Element {
       </Head>
       <body>
         <Main />
+        <div id='modal' />
```

Comment:
why do we need this change?

Reply:
Modal should appear outside of editorjs contaienr, that's why we are mounting modal next to main content

Reply:
can we mode this element inside a component where we are using emoji iteself?
...
<div id='modal' />
<Editor />
...

it is not good to add global div here for emoji

Reply:
what if I want to use emoji in comment textarea? will this root div cause a problem?

Reply:
No, it won't cause a problem. Because you can't have a multiple modals open simultaneously